app/views/pages/faq.html.erb

```diff
@@ -3,5 +3,140 @@
+            <p class="about_p">In terms of time, this estimate is based on a few factors, such as how many other users are in the queue and the size of your collection. You only need to click the “Analyze” button once. Don't worry: once your collection is done analyzing, we will send you an e-mail.</p>
+            <p class="about_p"><strong>Can I bring my own WARC files to the Cloud?</strong></p>
+            <p class="about_p">Right now the Archives Unleashed Cloud is only able to handle data by way of Archive-It subscriptions. As noted above, we are looking at ways of expanding this in the future.</p>
+            <p class="about_p">You are always welcome to use the <%= link_to('Archives Unleashed Toolkit', 'https://archivesunleashed.org/aut/', target: '_blank') %> to analyze your own WARC files, though this will require a bit of knowledge and patience with the command line. We have a <%= link_to('documentation', 'https://archivesunleashed.org/aut/', target: '_blank') %> and walk through instructions of how to use the Toolkit locally.</p>
+            <p class="about_p"><strong>What does the hyperlink diagram represent?</strong></p>
+            <p class="about_p">The interactive hyperlink diagram was designed to help users see their web archive collections at a glance, and visualizes the connections between the domains present within the collection.</p>
+            <p class="about_p">The diagram is based on social network theory, which visualizes the communication or connection relationships between entities (individuals, groups, organizations, institutions).</p>
+            <p class="about_p"><strong>I have the derivatives, but now what do I do?</strong></p>
+            <p class="about_p">Now comes the fun part of digging into your web archival collections. The derivatives give you a starting point for further analysis on your research questions. The way you approach analyzing the derivatives will largely depend on the content of the collections, topic, and what some of your research questions are, but here are a few ideas and resources to help get you started:</p>
+            <ul><strong><%=link_to('Cloud Learning Guides','/derivatives') %></strong>: thanks in part to Sarah McTavish and Ian Milligan, there are a number of guides that walk through using each derivative file with additional tools. </ul>
+            <ul><strong>Cloud Hackathon team projects</strong>: at each <%= link_to('#hackarchives', 'https://archivesunleashed.org/events/', target: '_blank') %> event, teams have the opportunity to work with the Archives Unleashed Toolkit and other resources to uncover the hidden gems in their collections. Have a look at some of the final projects to see what types of visualizations and processes were used for analysis (<%= link_to('Toronto', 'https://archivesunleashed.org/toronto/', target: '_blank') %>,<%= link_to('Vancouver', 'https://archivesunleashed.org/vancouver/', target: '_blank') %>).</ul>
```

Comment:
(Toronto,Vancouver)
->
(Toronto, Vancouver) 

adds space after comma in list